### PR TITLE
[NF-20097] Add the documentation for applicant details

### PR DESCRIPTION
**Pull request by zefta**
We're adding the documentation for the Applicant Details API endpoint, this endpoint allows you to edit the redirect_eligible column from the details table. Note that this endpoint is limited 1000 records per call.

Linked PR: https://github.com/Liaison-Intl/WebAdMIT/pull/7517

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/Liaison-Intl/prmd.git
-  revision: 8cb419a9b12bbf3356fd904a0628679630a85dfd
+  revision: 20b43706b0421b02021f49a5658e2f38fadb3194
```
Comment (zefta) [inline Gemfile.lock]: The gem has a bug which I fixed in https://github.com/Liaison-Intl/prmd/pull/1

File: schemata/applicant_details.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-04/hyper-schema",
+  "id": "/schemata/applicant_details",
+  "title": "Applicant Details",
+  "stability": "prototype",
+  "description": "The applicant details stores information about the applicant, only some of those information are editable.",
+  "links": [
+    {
+      "description": "Update a batch of Applicant Details, this endpoint is **limited to 1000 records per call**.",
```
Comment (Bunter) [inline schemata/applicant_details.json]: We should probably say **limited to 1000 Applicants per call** ?
Details are 1:1 and I think it is a little clearer. what do you think?

Comment (dquimper) [reply schemata/applicant_details.json]: 👍

**Comment by zefta** [comment]
@Bunter @dquimper I addressed your comments, let's not merge this until it's deployed to PROD otherwise the documentation will be live while the code is not deployed yet.

**Comment by cdarne** [comment]
Indeed 😅

**Pull request merged**